cli/cli-client/src/main/java/org/infinispan/cli/CLI.java

```diff
@@ -108,10 +108,15 @@ public final void run(String[] args) {
          }
          switch (command) {
             case "-c":
-               parameter = iterator.next();
+               if(iterator.hasNext()) {
```

Comment:
Not quite: the next element could be another switch.

Reply:
This is the same for other options (and server.sh as well)
If the parameter is missed the error message might become confusing

Reply:
No, because I want this to work for something like `-c -f file`